<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnvironDialog</class>
 <widget class="QWidget" name="EnvironDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>308</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Variable value:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="btnNewVariable">
     <property name="text">
      <string>New variable</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Variable name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <layout class="QVBoxLayout" name="veLayoutVarValues"/>
   </item>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="veLayoutEquals"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Set environment variables passed to all executed processes.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QVBoxLayout" name="veLayoutVarNames"/>
   </item>
   <item row="3" column="3">
    <layout class="QVBoxLayout" name="veLayoutRemoval"/>
   </item>
   <item row="4" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
